hack/cronjob-example.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: gcloud-export-example
+spec:
+  schedule: "@midnight"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+        spec:
+          restartPolicy: Never
+          containers:
+          - name: gcloud-sql-export
+            image: mintel/k8s-gcloud-export:initial
+            env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /sa/credentials.json
+            - name: GOOGLE_SQL_INSTANCE_NAME
+              value: my-db-instance-name
+            - name: GOOGLE_SQL_BACKUP_BUCKET
+              value: my-backup-bucket-name
+            - name: GOOGLE_SQL_BACKUP_BUCKET_PATH
+              value: some-folder/some-sub-folder
+            - name: DATABASE 
+              value: my-database
+            resources:
+              limits:
+                cpu: 200m
+                memory: 256Mi
+              requests:
+                cpu: 100m
+                memory: 64Mi
```

Comment:
Are these a bit high for something that just runs gcloud?

Reply:
So I did notice it took a long time to auth with `30m` cpu so I upped it to 100m. I can do some better testing on timings. 

The memory is perhaps high for sure.

Reply:
I decreased memory usage. 

I think CPU usage is fine as is -`100m` is not much! :)